# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/pixman/hotfix-vsx.patch
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- pixman-0.40.0/meson.build.vanilla	2022-03-08 18:05:22.771859334 +0100
+++ pixman-0.40.0/meson.build	2022-03-08 18:04:14.475855811 +0100
@@ -220,7 +220,7 @@
 
 use_vmx = get_option('vmx')
 have_vmx = false
-vmx_flags = ['-maltivec', '-mabi=altivec']
+vmx_flags = ['-maltivec', '-mabi=altivec', '-mvsx']
 if not use_vmx.disabled()
   if host_machine.cpu_family().startswith('ppc')
     if cc.compiles('''
